src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```

Comment:
4.[Checklist}Prefer `if` with `return` over `if else` to simplify later conditions.